### ISSUE TITLE
Can't install 

### ISSUE THREAD

**Issue by rizar** [bartvm/mimir#5]
Gives the following error: `error: unknown file type '.pyx' (from 'gzlog/gzlog.pyx')`. The full log is below:

```
╰─$ pip install --user .
Processing /home/rizar/Dist/mimir
Requirement already satisfied (use --upgrade to upgrade): pyzmq in /home/rizar/.local/lib/python2.7/site-packages (from mimir==0.1.dev1)
Requirement already satisfied (use --upgrade to upgrade): six in /home/rizar/.local/lib/python2.7/site-packages (from mimir==0.1.dev1)
Requirement already satisfied (use --upgrade to upgrade): simplejson in /usr/lib/python2.7/dist-packages (from mimir==0.1.dev1)
Installing collected packages: mimir
  Running setup.py install for mimir
    Complete output from command /usr/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip-ybA77E-build/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-nHIRCH-record/install-record.txt --single-version-externally-managed --compile --user --prefix=:
    running install
    running build
    running build_py
    creating build
    creating build/lib.linux-x86_64-2.7
    creating build/lib.linux-x86_64-2.7/mimir
    copying mimir/plot.py -> build/lib.linux-x86_64-2.7/mimir
    copying mimir/formatters.py -> build/lib.linux-x86_64-2.7/mimir
    copying mimir/utils.py -> build/lib.linux-x86_64-2.7/mimir
    copying mimir/remote.py -> build/lib.linux-x86_64-2.7/mimir
    copying mimir/serialization.py -> build/lib.linux-x86_64-2.7/mimir
    copying mimir/logger.py -> build/lib.linux-x86_64-2.7/mimir
    copying mimir/handlers.py -> build/lib.linux-x86_64-2.7/mimir
    copying mimir/__init__.py -> build/lib.linux-x86_64-2.7/mimir
    copying mimir/stream.py -> build/lib.linux-x86_64-2.7/mimir
    running build_ext
    building 'mimir.gzlog' extension
    error: unknown file type '.pyx' (from 'gzlog/gzlog.pyx')

    ----------------------------------------
Command "/usr/bin/python -c "import setuptools, tokenize;__file__='/tmp/pip-ybA77E-build/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record /tmp/pip-nHIRCH-record/install-record.txt --single-version-externally-managed --compile --user --prefix=" failed with error code 1 in /tmp/pip-ybA77E-build
/home/rizar/.local/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
You are using pip version 7.1.2, however version 8.0.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

**Comment by bartvm** [comment]
Works for me in a completely empty environment. Could you try upgrading `pip` to something newer?

**Comment by rizar** [comment]
The same error with the newest pip.

**Comment by rizar** [comment]
Resolved: too old version of setuptools (was 3.3, became 20.2.2)!
